hawkular-wildfly-monitor/src/main/assembly/module-assembly.xml

```diff
@@ -39,6 +39,14 @@
       <directoryMode>0755</directoryMode>
     </fileSet>
   </fileSets>
+  <files>
+    <file>
+      <source>${project.basedir}/src/main/assembly/subsystem.xml</source>
+      <outputDirectory>/${moduleDir}</outputDirectory>
+      <destName>subsystem-snippet.xml</destName>
+    </file>
+  </files>
+
 
   <dependencySets>
     <dependencySet>
```

Comment:
Does this affect in a any way the new feature pack build? We want to make sure this doesn't break or otherwise affect the feature pack and the ability of swarm to consume our feature pack. (see the feature pack module in our repo).